docs/internal-documentation/maintenance/ReleaseProcess-create-test-distribution.md

```diff
@@ -36,14 +36,15 @@ the `develop` branch and that your clone is up-to-date.
        git fetch --all
        git branch
 
+For Zonemaster-LDNS only - reset submodule (LDNS):
```

Comment:
For information, it's more an update of the submodule than a reset (the submodule is checked out to the right commit).

Reply:
Corrected. Please re-review.